### PR TITLE
try to solve page html structure

### PR DESCRIPTION
**Pull request by hilleer**

### CODE REVIEW

File: src/Components/html/Section.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 	constructor(props) {
 		super(props);
 	}
 	render() {
-		const { children } = this.props;
+		const { children, sectionClass } = this.props;
+		const classes = `section section--${sectionClass}`;
 		return (
-			<section className="section">
+			<section className={classes}>
```
Comment (gundlev) [inline src/Components/html/Section.js]: hvorfor ikke bare sætte string direkte ind?

File: src/Components/views/addMeal/AddMeal.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+export default class AddMeal extends Component {
+	render() {
+		const footerInfo = {
+			state: 'addMeal'
+		};
+		// const match = this.props.match.params.meal;
+		console.log('props: ', this.props);
+		const { params } = this.props;
+		const { meal } = params;
```
Comment (gundlev) [inline src/Components/views/addMeal/AddMeal.js]: if this is all you are extracting `const { meal } = this.props.params;` would be fine :)

File: src/Components/views/home/Home.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		};
+		return (
+			<Page footerInfo={footerInfo}>
+				<PageBody>
+					<Section sectionClass="home">
+						<HorizontalLine />
+						<p className='menu-category--title-p'>
+							<span className='menu-category--title-span'>sandwiches</span>
+						</p>
+						<MenuItem
```
Comment (gundlev) [inline src/Components/views/home/Home.js]: You will hvae these in an array, so you can do this instead: 
```
{
	menuItems.map((item, index) =>
		<div>
			<p className='menu-category--title-p'>
				<span className='menu-category--title-span'>sandwiches</span>
			</p>
			<MenuItem
				title={item.title}
				description={item.description}
				price={item.price}
			/>
			<HorizontalLine />
		</div>
	)
}
```

File: src/Components/views/addMeal/AddMeal.js (hunk 2 above)
Comment (hilleer) [reply src/Components/views/addMeal/AddMeal.js]: Yeah, agreed. Was done this way as I expected this.props to hold additional properties at some point, probably.

**Pull request merged**